cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -973,6 +973,86 @@ def submit_aces(
             lifespan=lifespan,
         )
 
+    def submit_cb(
+        self,
+        target: str,
+        shots: int,
+        circuits: cirq.Circuit,
+        num_channels: int,
+        num_sequences: int,
+        depths: list[int],
+        method: str | None = None,
+        noise: str | cirq.NoiseModel | None = None,
+        error_prob: float | tuple[float, float, float] | None = None,
+    ) -> str:
+        """Submits the jobs to characterize `target` through the Cycle Benchmarking protocol.
+
+        The protocol in detail can be found in: https://arxiv.org/abs/1902.08543.
+
+        Args:
+            target: The device target to characterize.
+            shots: How many shots to use per circuit submitted.
+            circuits: The process circuit to use in the protocol.
+            num_channels: The number of random Pauli decay channels to approximate error.
+            num_sequences: Number of circuits to generate per depth.
+            depths: Lists of depths representing the depths of Cycle Benchmarking
+                circuits to generate.
+            method: Optional method to use in device submission (e.g. "dry-run").
+            noise: Noise model to simulate the protocol with. It can be either a string or a
+                `cirq.NoiseModel`. Valid strings are "symmetric_depolarize", "phase_flip",
+                "bit_flip" and "asymmetric_depolarize".
+            error_prob: The error probabilities if a string was passed to `noise`.
+                * For "asymmetric_depolarize", `error_prob` will be a three-tuple with the error
+                rates for the X, Y, Z gates in that order. So, a valid argument would be
+                `error_prob = (0.1, 0.1, 0.1)`. Notice that these values must add up to less than
+                or equal to 1.
+                * For the other channels, `error_prob` is one number less than or equal to 1, e.g.,
+                `error_prob = 0.1`.
+        Returns:
+            A string with the job id for the Cycle Benchmarking job created.
+
+        Raises:
+            ValueError: If the target or noise model is not valid.
+            SuperstaqServerException: If the request fails.
+        """
+
+        noise_dict: dict[str, object] = {}
+        if isinstance(noise, str):
+            noise_dict["type"] = noise
+            noise_dict["params"] = (
+                (error_prob,) if isinstance(error_prob, numbers.Number) else error_prob
+            )
+        elif isinstance(noise, cirq.NoiseModel):
+            noise_dict["cirq_noise_model"] = cirq.to_json(noise)
+
+        serialized_circuits = css.serialization.serialize_circuits(circuits)
+
+        return self._client.submit_cb(
+            target,
```

Comment:
```suggestion
            self._resolve_target(target),
```
(so this falls back on `default_target`)